vernac/vernacentries.ml

```diff
@@ -522,13 +523,17 @@ let start_lemma_com ~program_mode ~poly ~scope ~kind ?hook thms =
   in
   start_lemma_with_initialization ?hook ~poly ~scope ~kind evd ~udecl recguard thms snl
 
-let vernac_definition_hook ~local ~poly = let open Decls in function
+let vernac_definition_hook ~canonical_instance ~local ~poly = let open Decls in function
 | Coercion ->
   Some (Class.add_coercion_hook ~poly)
-| CanonicalStructure ->
-  Some (DeclareDef.Hook.(make (fun { S.dref } -> Canonical.declare_canonical_structure ?local dref)))
 | SubClass ->
   Some (Class.add_subclass_hook ~poly)
+| CanonicalStructure ->
```

Comment:
Is it the case now that `Canonical` is the same as `#[canonical] Definition`

Reply:
Yes, see the test suite change